free-courses-en.md

```diff
@@ -187,7 +188,10 @@
 * [Docker Tutorial for Beginners](https://www.youtube.com/watch?v=fqMOX6JJhGo) - FreeCodeCamp
 * [Learn Docker, Container Runtimes, Builders and Registries](https://www.katacoda.com/courses/container-runtimes) - Katacoda
 
-
```

Comment:
Please add back this line break. There should be two line breaks above a new section.